EstateReporting.IntegrationTests/Common/TestingContext.cs

```diff
@@ -14,6 +14,8 @@ public class TestingContext
     {
         #region Fields
 
+        public DateTime DateToUseForToday;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/stuart_ferguson1/EstateReporting/pullRequest?prid=6346796)